docs/learn-about-neptune/ui.rst

```diff
@@ -0,0 +1,137 @@
+Working with the Neptune UI
+===========================
+
+The Neptune client is a browser-enabled app that lets you visualize and browse experiments.
+
+
+   .. image:: /_static/images/core-concepts/experiment_general_view.png
+      :target: /_static/images/core-concepts/experiment_general_view.png
+      :alt: Experiments view
+
+
+Supported browsers
+------------------
+
+Neptune supports Chrome and Firefox, on all major operating systems.
+
+Log in to the app
+-----------------
+1. In a web browser, navigate to ``ui.neptune.ai`` or, in the case of an on-prem deployment, to your Neptune instance address.
+2. Click **Login**.
+
+.. image:: /_static/images/core-concepts/login.png
+   :target: /_static/images/core-concepts/login.png
+   :alt: Login screen
+   :width: 350
+
+
+3. Type in your credentials and click **Sign in**.
+
+Experiments View
+----------------
+
+
+The **Experiments** tab  displays all the experiments in a specific Project in table form.
+
+Filter Experiments
+^^^^^^^^^^^^^^^^^^
+You can perform the simplest filtering by typing into the search fields:
+
+   .. image:: /_static/images/core-concepts/search_fields.png
+      :target: /_static/images/core-concepts/search_fields.png
+      :alt: Experiment search
+
+
+You can also use the `Neptune Query Language <nql.html>`_ to filter experiments for more advanced criteria.
+
+Organize Experiments
+^^^^^^^^^^^^^^^^^^^^
+
+There are several ways to organize your experiments:
+
+- **Tags**: You can create tag(s), which you assign to experiments. Later, you can quickly filter by these tags.
+
+   .. image:: /_static/images/core-concepts/tag_chooser.png
+      :target: /_static/images/core-concepts/tag_chooser.png
+      :alt: Tag chooser
+      :width: 250
+
+- **Custom views**: You can create a custom view of the dashboard, for example, by filtering selected rows, columns and columns order, sorting and width. Then you can save the view and quickly return to it by selecting the desired view from the list of views.
+
+   .. image:: /_static/images/core-concepts/view_list.png
+      :target: /_static/images/core-concepts/view_list.png
+      :alt: View list
+      :width: 400
+
+- **Choosing columns**: Columns in the dashboard are configurable. All types of data logged to Neptune can be seen as columns in the dashboard, for example parameters, metrics or tags.
+
+- **Grouping**: You can group experiments by one or more column(s). The dashboard displays the selected columns, allowing you to make in-group and across-groups analysis of the experiments.
```

Comment:
Maybe we can write here that each group is represented by its first experiment (according to sorting order)?
And also, that each group (after opening it) shows at most 10 experiments and all experiments can be viewed by clicking `Show all`.